### ISSUE TITLE
Disable hotkey altogether

### ISSUE THREAD

**Issue by raulgabor** [annotorious/annotorious-openseadragon#192]
Hi,

Is there a way for the hotkey to be totally disabled so that annotations can be drawn only from buttons actions? Or any workaround for it to disable it? I've searched through the documentation and I didn't see an option for that. Any help is appreciated. Thanks!

**Comment by rsimon** [comment]
Indeed, I don't think that's currently possible I'm afraid. You could try your luck with setting the hotkey to inverted, and then assigning some unused key value (or even invalid...) That might serve as a workaround. (But I haven't tried it.)

**Comment by kaleemnisar478** [comment]
At viewer add handler for canvas-key event that prevents default action
`viewer.addHandler('canvas-key', handlePreventDefault);
  const handlePreventDefault = (event: CanvasKeyEvent) => {
    event.preventDefaultAction = true;
  };
`
